hedera-node/src/main/java/com/hedera/services/utils/invertible_fchashmap/IdentifyableUniqueToken.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ‌
+ * Hedera Services Node
+ * Copyright (C) 2018 - 2020 Hedera Hashgraph, LLC
```

Comment:
2021, the whole licence should be beneath the package

Reply:
This file is a leftover, which returned after my revert. Will be removed.